frontend/src/components/experiments_table_cell/ResultName.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, FormGroup, Input } from 'reactstrap';
+
+import {
+  getRelativeResultPathName,
+  displayResultNameFull,
+  truncate
+} from '../../utils';
+
+class ResultName extends React.Component {
```

Comment:
[trivial?] Result names are difficult to edit. Changes to the result name are discarded when the result is reloaded via the API.